### DISCUSSION TITLE
[Feature] Kodi app

### DISCUSSION THREAD

**Discussion by lmgithu** [immich-app/immich#10427] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to propose a Kodi / OSMC app. There are a ton of devices that use Kodi as a media player, it has a robust addon librady already. Currently there is no official way to view Immich photos on TV, with this we could easily have it. Kodi is also open source, so it would fit beautifully into Immich’s approach too.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by jensfriisnielsen** [comment]
This is a duplicate of https://github.com/immich-app/immich/discussions/9388#discussion-6645460
